env/numerical_compliance.asciidoc

```diff
@@ -1357,197 +1357,199 @@ Profile>> table when operating in the full profile, and as defined in the
 instructions for Embedded Profile>> table when operating in the embedded
 profile.
 
-// TODO: Eventually add appropriate bolding to the instructions in the table below.
-
 [[ulp_values_for_math_instructions_with_fast_relaxed_math]]
 .ULP Values for Single Precision Math Instructions with _-cl-unsafe-math-optimizations_
 [width="100%",cols="30%,70%",options="header"]
 |====
-| *SPIR-V Instruction*
+| *Function*
 | *Minimum Accuracy*
 
-| *OpFDiv* for 1.0 / _x_
-| \<= 2.5 ulp for x in the domain of 2^-126^ to 2^126^ for the full profile,
-  and \<= 3 ulp for the embedded profile.
+| *1.0 / _x_*
```

Comment:
This is now the only table entry which doesn't reference a SPIR-V instruction, is that intentional?

Reply:
Good catch, this was not intended.  I added back `OpFDiv` to this description.